### PR TITLE
Yasmin mp telemetry 2.0

### PR DESCRIPTION
**Pull request by benjamin-confino**
Part of #27793

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
Could not get resource 'https://eu-public.artifactory.swg-devops.com/artifactory/wasliberty-open-liberty/org/eclipse/transformer/org.eclipse.transformer.bnd.analyzer/0.5.0/org.eclipse.transformer.bnd.analyzer-0.5.0.pom'.
            > Could not GET 'https://eu-public.artifactory.swg-devops.com/artifactory/wasliberty-open-liberty/org/eclipse/transformer/org.eclipse.transformer.bnd.analyzer/0.5.0/org.eclipse.transformer.bnd.analyzer-0.5.0.pom'. Received status code 503 from server: Service Temporarily Unavailable
            
Using my browser I can download it from na but not eu.

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by Yas-Franco** [comment]
#libby

**Review by Yas-Franco — DISMISSED** [review]

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/bnd.bnd]: Copyright needs correcting

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/build.gradle [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/build.gradle]: Copyright needs correcting

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry11TCKServer/bootstrap.properties [hunk 3]
```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2023 IBM Corporation and others.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry11TCKServer/bootstrap.properties]: Copyright needs correcting

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry11TCKServer/server.xml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<!--Copyright (c) 2023 IBM Corporation and others. All rights reserved.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry11TCKServer/server.xml]: Copyright needs correcting

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/pom.xml [hunk 5]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2023 IBM Corporation and others. All rights reserved.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/pom.xml]: Copyright needs correcting

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml [hunk 6]
```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Copyright (c) 2023 IBM Corporation and others. All rights reserved.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml]: Copyright needs correcting

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<!--Copyright (c) 2023 IBM Corporation and others. All rights reserved.
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml]: Copyright needs correcting

File: dev/io.openliberty.mpTelemetry.2.0.thirdparty/bnd.bnd [hunk 8]
```diff
@@ -0,0 +1,53 @@
+#*******************************************************************************
+# Copyright (c) 2022, 2023 IBM Corporation and others.
```
Comment (abutch3r) [inline dev/io.openliberty.mpTelemetry.2.0.thirdparty/bnd.bnd]: Copyright needs correcting as should just be 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/pom.xml [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.microprofile.telemetry</groupId>
+    <artifactId>tck.runner</artifactId>
+    <version>1.0</version>
+    <packaging>pom</packaging>
+    <name>MicroProfile Telemetry 1.1 TCK Runner</name>
```
Comment (abutch3r) [inline dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/tckRunner/pom.xml]: Needs updating to say 2.0 instead of 1.1.

**Comment by abutch3r** [comment]
Once the name of 2.0 TCK is fixed, happy to approve

**Review by abutch3r — APPROVED** [review]
LGTM

**Comment by benjamin-confino** [comment]
#libby

**Pull request merged**